README.md

```diff
@@ -2,7 +2,127 @@
 
 # Metamath set.mm repository
 
-This is a set of Metamath source files (databases) for logic and set theory.
+This is a collection of databases that specify mathematical axioms and
+rigorous formal proofs of various mathematical theorems
+derived from those axioms.
+This collection is specified and derived using the Metamath system.
+A "database" in Metamath parlance is a collection of one or more text files.
 
-File "set.mm" uses classical logic and ZFC set theory.
-File "iset.mm" uses intuitionistic logic.
+This collection is being actively improved by an international community.
+Please join us!
+
+The two largest databases in this collection are "set.mm" and "iset.mm".
+The "set.mm" database is one of the largest
+collections of formally-verified mathematics in the world (e.g., it
+has completed many challenge theorems in the
+[Formalizing-100 Theorems](https://www.cs.ru.nl/~freek/100/) challenge list).
+It is also one of the most rigorously verified collection of
```

Comment:
As already suggested by others, references to http://us.metamath.org/mpeuni/mmset.html (or http://us.metamath.org/mpeuni/mmtheorems.html?) and http://us.metamath.org/ileuni/mmil.html (or http://us.metamath.org/ileuni/mmtheorems.html?) should be added here.

Reply:
Agreed!

I think the right starting points are http://us.metamath.org/mpeuni/mmset.html  and http://us.metamath.org/ileuni/mmil.html - they provide context & then link on to their table of contents (as well as providing other useful starting points.